backend/layers/persistence/persistence.py

```diff
@@ -596,23 +596,35 @@ def get_dataset_version(self, dataset_version_id: DatasetVersionId, get_tombston
                     return None
             return self._hydrate_dataset_version(dataset_version)
 
+    def get_most_recent_active_dataset_version(self, dataset_id: DatasetId) -> Optional[DatasetVersion]:
+        """
+        Returns the must recently active Dataset version for a canonical dataset_id
```

Comment:
```suggestion
        Returns the most recently active Dataset version for a canonical dataset_id
```

What does it mean if it returns None?

Reply:
No such canonical Dataset. Similar to [the way this method works](https://github.com/chanzuckerberg/single-cell-data-portal/blob/d09ddc97937aad4a9a345c0c12dcc2efb7fdc0d0/backend/layers/persistence/persistence.py#L185-L187).